include/todo_lib/TodoModel.h

```diff
@@ -0,0 +1,34 @@
+#ifndef TODOMODEL_H
+#define TODOMODEL_H
+
+#include <string>
+
+class TodoModel {
+public:
+    // attributes
+    unsigned int id;
```

Comment:
did you check if there’s any Unique ID data type built in, so that we can avoid using unsigned int?

Reply:
Guess this is not resolved yet?